### ISSUE TITLE
A confusion about the coarse_matching

### ISSUE THREAD

**Issue by BaochunHe** [zju3dv/LoFTR#261]
在cparse_matching.py文件的第245行，感觉index是反过来的，为什么是
mkpts0_c = torch.stack( [i_ids % data['hw0_c'][1], i_ids // data['hw0_c'][1]], dim=1) * scale0，
而不是
mkpts0_c = torch.stack( [ i_ids // data['hw0_c'][1]，i_ids % data['hw0_c'][1]], dim=1) * scale0

**Comment by chen9run** [comment]
我也是相同的感觉，这个地方感到十分困惑，你有做尝试嘛或者有什么见解嘛
